config/files/usr/etc/pam.d/password-auth

```diff
@@ -0,0 +1,62 @@
+# secureblue password pam service
+# this is a mirror of the default service in fedora with libpam-authramp and documentation added.
```

Comment:
instead of mirroring the existing config, augment the config using `sed` or `>>`

Otherwise, we'll constantly have to ensure that upstream changes are pulled in, which would be unmaintainable.